locales/en-human.json

```diff
@@ -261,11 +261,16 @@
   "HIDE_PAIRING_CODE"  : "Hide Pairing Code",
   "SHOW_PAIRING_CODE"  : "Show Pairing Code",
   "PAIRING_CODE_EXPLAIN": "Scan the code <i>(click on 'Show Pairing Code')</i> with your Blockchain Wallet (iOS or Android) for a seamless connection to your wallet.",
+  "SHOW_PAIRING_CODE_FAIL" : "Failed to load pairing code",
   "DOWNLOAD_THE": "Download the",
   "IOS_APP_HERE": "iOS app here",
   "AND_THE": "and the",
   "ANDROID_APP_HERE": "Android app here",
   "MOBILE" : "Mobile",
+  "CONFIRM_DISABLE_ALIAS" : "<i class='ti-alert security-red mrs'></i>Remove Wallet Alias?<b><br/><br/><small>Once your alias is removed, you will not be able to use it to log in. This cannot be undone. Make sure you write down your Wallet Identifier before continuing.<br/><br/>Wallet ID: <b>{{::id}}</b></small>",
```

Comment:
You're right, it's better to have this html in a partial. We have a bunch of warning alerts that follow this format, will clean them up and add to the PR.